sdk/containerregistry/Azure.Containers.ContainerRegistry/src/autorest.md

```diff
@@ -4,7 +4,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-    -  https://raw.githubusercontent.com/Azure/azure-rest-api-specs/dbd1dccfe2802010a0abc76e250ccbd55f4f2837/specification/containerregistry/data-plane/Azure.ContainerRegistry/preview/2019-08-15/containerregistry.json
+    -  https://github.com/jeremymeng/azure-rest-api-specs/blob/96bdb14b80d5542ea6982cd766be19e7c8c58c86/specification/containerregistry/data-plane/Azure.ContainerRegistry/preview/2019-08-15-preview/containerregistry.json
```

Comment:
Should this be pointing to "Azure" now, or is this just temporary?

Reply:
We'll point it to the readme.md in "Azure" once the PR is merged.